packages/css/src/less/core/_scaffolding.less

```
@@ -241,11 +241,11 @@ ul {
 
 dt,
 dd {
-  line-height: @line-height-base;
+  line-height: var(--line-height-body);
 }
 
 dt {
-  font-size: @font-size-small;
+  font-size: var(--font-size-12);
```

I think this should be 14. We want to avoid using fonts smaller than 14 for essential information, and we use `dt` in places like the [DefinitionList](https://transferwise.github.io/neptune-web/branch/update-typography-styles/components/content/DefinitionList), where the label is essential.

```suggestion
  font-size: var(--font-size-14);
```